miasm2/analysis/binary.py

```diff
@@ -209,6 +209,16 @@ def parse(self, data, vm=None, addr=0, apply_reloc=False, **kwargs):
             # Add known symbols (vm_load_elf already does it)
             fill_loc_db_with_symbols(self._executable, self.loc_db, addr)
 
+            if hasattr(self._executable.sh, 'dynsym'):
```

Comment:
This should rather be done in `fill_loc_db_with_symbols`

Reply:
I agree with you: I moved the code

Reply:
I agree with you: I moved the code